linux_os/guide/services/ssh/ssh_server/sshd_use_approved_kex_ordered_stig/tests/comment.fail.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 # platform = multi_platform_ol,multi_platform_rhel,multi_platform_sle,multi_platform_slmicro,multi_platform_ubuntu,multi_platform_almalinux
+{{% if product in ['ol8', 'rhel8']  %}}
```

Comment:
We should remove this extra blank line. This applies to all of the files in this PR.